### PR TITLE
remove deleted channels from discord's README

### LINKED ISSUE #1079: Invalid-Channel Name in ReadMe File for EddieHub Discord Android UI
**Description**

Channel Name is removed but still referenced so it is being referenced as Invalid Channel Name in Discord Android UI.

**Screenshots**

Screenshot_2021-03-14-16-39-25-476_com discord

Also, I think this bug or issue is just in the Discord App on Android as in the Windows Discord it is being shown as <#deleted-channel>

### PR DESCRIPTION
**Pull request by kcoder63**
closes #1079 

#### What does this PR do?

removed Chilling section, cloud and chatting channels, moved heathfitness channel to text channels and renamed livestream to youtube.

#### Description of Task to be completed?
update the discord's readme

### CODE REVIEW

**Review by srinath1412001 — COMMENTED** [review]
LGTM!

File: discord-readme.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ TEXT CHANNELS: These consist of the server's main text channels, that are for di
 
 -> #standup: everyday of code, give us your standup updates! What you did yesterday,what you plan to do today and any blockers?
 
+-> #healthfitness: health and fitness caus you want to get those bizeps
```
Comment (galahad42) [inline discord-readme.md:44]:
```suggestion
-> #healthfitness: health and fitness cause you want to get those biceps

```

**Review by galahad42 — CHANGES_REQUESTED** [review]
Minor grammatical errors fixed.

File: discord-readme.md (hunk 1 above)
Comment (galahad42) [reply discord-readme.md:44]: Actually a better sentence would be " health and fitness so you can get those biceps"

Comment (kcoder63) [reply discord-readme.md:44]: Yep will change it

**Comment by srinath1412001** [comment]
> Minor grammatical errors fixed.

Oh.. I missed them. Nice work!

**Review by schmelto — COMMENTED** [review]
LGTM

**Review by SamDev98 — DISMISSED** [review]
LGTM

**Review by SamDev98 — COMMENTED** [review]
Just one thing, should we add https://github.com/EddieHubCommunity/support/blob/main/discord-readme.md to it as well? So people know what to raise issue/PR against?

**Comment by kcoder63** [comment]
Oh yes, thank you for reminding, that was the idea.

**Comment by SashoStoichkov** [comment]
@kcoder63 what do you think about adding the headings as well:

ex.
```
DAILY:
-> # standup
-> # achievements
-> # healthfittness
```

**Comment by kcoder63** [comment]
yes,can add that

**Comment by galahad42** [comment]
I think @SashoStoichkov  meant `Channel Category` --> `Channel list` type structure.

**Comment by SashoStoichkov** [comment]
Yep 👍🏻

**Review by SashoStoichkov — APPROVED** [review]
Thank you for the contribution 👍🏻 LGTM

**Review by galahad42 — APPROVED** [review]
LGTM 👍

**Review by SamDev98 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Review by meenal21 — APPROVED** [review]
LGTM!

**Review by SamDev98 — APPROVED** [review]

**Review by AllanRegush — APPROVED** [review]
LGTM :+1:

**Comment by AllanRegush** [comment]
Thanks to everyone who reviewed.

**Comment by eddiejaoude** [comment]
Great collaboration everyone 💪

**Pull request merged**